account_invoice_base_invoicing_mode/readme/ROADMAP.rst

```diff
@@ -0,0 +1,7 @@
+On the version 13.0 PR it has been discussed to rename the modules like this:
+
+    * account_invoice_base_invoicing_mode -> partner_invoicing_mode
```

Comment:
why not renamed then?

Reply:
I think it is not up to me to decide this...